### PR TITLE
Update it_IT.lang

### PR DESCRIPTION
**Pull request by TheVikingWarrior**

### CODE REVIEW

File: lang/it_IT.lang [hunk 1]
```diff
@@ -187,4 +189,5 @@ myst.symbol.WeatherSlow.name=Tempo atmosferico lento
 myst.symbol.WeatherSnow.name=Neve eterna
 myst.symbol.WeatherStorm.name=Tempesta eterna
 myst.symbol.Villages.name=Villaggi
+
```
Comment (XCompWiz) [inline lang/it_IT.lang]: Can you remove this added line before the last symbol? :)

**Comment by TheVikingWarrior** [comment]
Guess i'm colour blind... :(

**Comment by XCompWiz** [comment]
Great! Now can you squash it back to just one commit? :P

**Comment by Vexatos** [comment]
@XCompWiz Many people do their translations in-browser so squashing would require quite a bit of effort, and considering this is a repo purely for lang files, you might want to consider not requiring commit squashing.

**Comment by XCompWiz** [comment]
If the squashing is too much effort then I'd ask a new PR be made. :)
I'm picky, I know, but I'm going to stick to my standards.

**Comment by XCompWiz** [comment]
I don't think it existed before, but GitHub has a "Squash and Merge" feature now.  Now I get to have my standards AND you guys can take it easy. Win win. :D

**Pull request merged**